WcaOnRails/app/views/competitions/index.html.erb

```diff
@@ -1,17 +1,139 @@
 <% provide(:title, 'Competitions') %>
 
 <div class="container">
-  <h1><%= yield(:title) %></h1>
-
-  <%= link_to "New competition", new_competition_path %>
-  <table class="table">
-    <% @competitions.each do |competition| %>
-      <tr>
-        <td><%= competition.id %></td>
-        <td><%= competition.name %></td>
-        <td><%= link_to "Edit", admin_edit_competition_path(competition) %></td>
-        <td><%= link_to "Results", "/results/competition.php?competitionId=#{competition.id}" %></td>
-      </tr>
+  <h2><%= yield(:title) %></h2>
+
+  <div id="scroller-anchor"></div>
+  <div id="comp-query" class="col-md-12">
+    <% if params[:display] == "List" %>
+      <ul>
```

Comment:
I'm starting to think that these will only be useful on mobile, because on big screens the 3 lists will be very close.

I'm not very good with icons...if you have suggestions, I'll be happy to try.